src/transformers/models/auto/image_processing_auto.py

```diff
@@ -411,6 +413,10 @@ def from_pretrained(cls, pretrained_model_name_or_path, *inputs, **kwargs):
         use_fast = kwargs.pop("use_fast", None)
         trust_remote_code = kwargs.pop("trust_remote_code", None)
         kwargs["_from_auto"] = True
+        default_image_processor_filename = (
+            "config.json" if is_timm_checkpoint(pretrained_model_name_or_path) else IMAGE_PROCESSOR_NAME
+        )
```

Comment:
For the timm models, the processing information is contained within the `config.json` file e.g. https://huggingface.co/timm/resnet50.a1_in1k/blob/main/config.json